pkg/term/windows/console.go

```diff
@@ -0,0 +1,59 @@
+// +build windows
+
+package windows
+
+import (
+	"io"
+	"os"
+	"syscall"
+)
+
+// ConsoleStreams, for each standard stream referencing a console, returns a wrapped
+// version that handles ANSI character sequences.
+func ConsoleStreams() (stdIn io.ReadCloser, stdOut, stdErr io.Writer) {
+	if IsConsole(os.Stdin.Fd()) {
+		stdIn = newAnsiReader(syscall.STD_INPUT_HANDLE)
+	} else {
+		stdIn = os.Stdin
+	}
+
+	if IsConsole(os.Stdout.Fd()) {
+		stdOut = newAnsiWriter(syscall.STD_OUTPUT_HANDLE)
+	} else {
+		stdOut = os.Stdout
+	}
+
+	if IsConsole(os.Stderr.Fd()) {
+		stdErr = newAnsiWriter(syscall.STD_ERROR_HANDLE)
+	} else {
+		stdErr = os.Stderr
+	}
+
+	return stdIn, stdOut, stdErr
+}
+
+// GetHandleInfo returns file descriptor and bool indicating whether the file is a console.
+func GetHandleInfo(in interface{}) (uintptr, bool) {
+	switch t := in.(type) {
```

Comment:
it's common to reuse the same variable name (creating a new scope). So `switch stream := stream.(type) {`